### PR TITLE
add vrack and load balancer floating IP limitations

### PR DESCRIPTION
**Pull request by yomovh**

### CODE REVIEW

File: pages/public_cloud/public_cloud_network_services/known-limits/guide.en-gb.md [hunk 1]
```diff
@@ -4,6 +4,19 @@ excerpt: 'Requirements and limits to respect'
 updated: 2023-11-14
 ---
 
+## Vrack and Public Cloud project
+
+For a given Public Cloud project, you can attach only one vrack. If you wish to build private connectivity between 2 (or more) public cloud projects, you should attach the same vrack to those public cloud projects.
+
+## Load Balancer Floating IP in OVHcloud Control panel
+
+Currently the floating IP associated to a Load Balancer is missing in the Load Balancer details page on OVHcloud Control panel. You can find it in [Horizon](../../compute/introducing_horizon/), in [Openstack CLI](../../compute/prepare_the_environment_for_using_the_openstack_api/) by doing `openstack floating ip list` and `openstack loadbalancer list` or through OVHcloud API.
```
Comment (OVH-Team-Guides) [inline pages/public_cloud/public_cloud_network_services/known-limits/guide.en-gb.md]:
```suggestion
Currently the floating IP associated to a Load Balancer is missing in the Load Balancer details page on OVHcloud Control panel. You can find it in [Horizon](/pages/public_cloud/compute/introducing_horizon), in [Openstack CLI](/pages/public_cloud/compute/prepare_the_environment_for_using_the_openstack_api) by doing `openstack floating ip list` and `openstack loadbalancer list` or through OVHcloud API.
```
@yomovh , please use the standard link format for guides links

**Comment by OVH-Team-Guides** [comment]
@yomovh , please check the FR version

**Comment by yomovh** [comment]
Proof read FR done

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @yomovh

**Pull request merged**